website/docs/for-android/quick-start.md

```diff
@@ -147,110 +162,20 @@ Open Android Studio and place the following script at the bottom of your module
-
-<Tabs
-defaultValue="nix"
-values={[
-{ label: 'MacOS/Linux', value: 'nix', },
-{ label: 'Windows', value: 'win', },
-]}>
-<TabItem value="nix">
-
-```groovy title=build.gradle
-
-// ...
-
-// Make the preBuild task depend on getLiveUpdate so it runs every build
-preBuild.dependsOn 'getLiveUpdate'
-
-tasks.register('getLiveUpdate') {
-    doLast {
-        String appId = "0ee57266"
-        String channel = "PRODUCTION"
-        String portalName = "MY_FIRST_PORTAL"
-
-        String assetPath = rootDir.getPath() + '/app/src/main/assets/' + portalName
-        if (new File(assetPath).exists()) {
-            // If the web app already exists, replace with fresh download
-            project.exec {
-                commandLine 'rm', '-rf', assetPath
-            }
-            project.exec {
-                commandLine 'mkdir', '-p', assetPath
-            }
-        } else {
-            project.exec {
-                commandLine 'mkdir', '-p', assetPath
-            }
-        }
-
-        project.exec {
-            commandLine 'ionic-cloud', 'live-update', 'download', '--config=.ionic-cloud.yaml', '--app-id', appId, '--channel-name', channel, '--zip-name', portalName + '.zip'
-        }
-        project.exec {
-            commandLine 'unzip', portalName + '.zip', '-d', assetPath
-        }
-        project.exec {
-            commandLine 'rm', portalName + '.zip'
-        }
-    }
-}
-```
-
-</TabItem>
-<TabItem value="win">
-
-```java title=build.gradle
-
-// ...
 
-// Make the preBuild task depend on getLiveUpdate so it runs every build
-preBuild.dependsOn 'getLiveUpdate'
+```groovy title="build.gradle"
+//Make the preBuild task depend on syncPortals so it runs every build
+preBuild.dependsOn 'syncPortals'
 
-tasks.register('getLiveUpdate') {
+tasks.register('syncPortals') {
     doLast {
-        String appId = "0ee57266"
-        String channel = "PRODUCTION"
-        String portalName = "MY_FIRST_PORTAL"
-
-        String assetPath = rootDir.getPath() + '\\app\\src\\main\\assets\\' + portalName
-        if (new File(assetPath).exists()) {
-            // If the web app already exists, replace with fresh download
-            project.exec {
-                commandLine 'cmd', '/c', 'rm', '-r', assetPath
-            }
-            project.exec {
-                commandLine 'cmd', '/c', 'mkdir', assetPath
-            }
-        } else {
-            project.exec {
-                commandLine 'cmd', '/c', 'mkdir', assetPath
-            }
-        }
-
-        project.exec {
-            commandLine 'cmd', '/c', 'ionic-cloud', 'live-update', 'download', '--config=".ionic-cloud.yaml"', '--app-id', appId, '--channel-name', channel, '--zip-name', portalName + '.zip'
-        }
-        project.exec {
-            commandLine 'cmd', '/c', 'unzip', portalName + '.zip', '-d', assetPath
-        }
         project.exec {
-            commandLine 'cmd', '/c', 'rm', portalName + '.zip'
+            commandLine 'portals', 'sync'
```

Comment:
This still needs to know the location of the asset path

Reply:
It does through `.portals.yaml`:

```yaml
 - app-id: 11a0971f
   channel: production
   directory-name: src/main/assets/featured_products
```

Reply:
Unless someone has tested on a windows machine otherwise. I know it doesn't show up in the PATH on mac though.

Reply:
I have it working in another app. I do have it set on the PATH though, maybe I should call that out.

Reply:
> It does through `.portals.yaml`:
> 
> ```yaml
>  - app-id: 11a0971f
>    channel: production
>    directory-name: src/main/assets/featured_products
> ```

That's not really a portable example between iOS and android though. That would require the start dir on iOS be "src/main/assets/featured_products" if you're using the same config between ios and android

Reply:
> I have it working in another app. I do have it set on the PATH though, maybe I should call that out.

I don't know much about windows, but unless you're putting the portals cli in /usr/bin /usr/local/bin or one of the other default directories macOS puts in PATH it's not going to be picked up. I ran into this myself when getting trying to update portals-ecommerce-demo

Reply:
@Steven0351 - Finally had time to test this out:

1. The same path for Windows and *nix will work in `.portals.yaml`.
2. I figured it's good enough™️ to have a note mentioning the user needs to have the Portals CLI as part of their PATH to run the `build.gradle` task. 

I've verified the Android build task works on both Mac and Windows -- provided it's part of the PATH.

Reply:
Where is your portals cli located on your machine? Because from what I experienced if it is not in a folder that is part of the default macOS $PATH then it will not get picked up because of how the JVM forks. I was unable under any circumstances able to get it to pick up a $PATH environment variable that I set either when running the gradle task directly or through Android Studio. So, I feel like when people read this:
```
:::note
The Portals CLI must be registered as part of the `PATH` to run this command successfully.
:::
```
That they will think they can manipulate the PATH environment variable to get it to pick up, when it probably won't.

Reply:
Interesting, I just added it to my PATH in `.zshrc`:

```
export JAVA_HOME=/Applications/Android\ Studio.app/Contents/jbr/Contents/Home
export VOLTA_HOME="$HOME/.volta"
export PORTALS_PATH="/opt/homebrew/opt/portals"
export STUDIO_PATH="/Applications/Android Studio.app/Contents/MacOS"
export PATH="$VOLTA_HOME/bin:$PORTALS_PATH:$STUDIO_PATH:$PATH"
```

Reply:
Hm, are you opening Android Studio via the command line?

Reply:
Hey so I tested fresh on Mac and Windows and both were picked up fine by Android studio on both machines. I installed via Brew on Mac and when I restarted Android Studio it was picking it up, and on Windows I followed the instructions on the download page for the Portals CLI and put it in a folder in my user dir, and added that to my system PATH in Windows, then restarted Android Studio and it finds it.